documentation/jetty/modules/programming-guide/pages/server/http.adoc

```diff
@@ -1510,6 +1510,43 @@ Use of this class is less flexible than using the <<handler-use-rewrite,`Rewrite
 include::code:example$src/main/java/org/eclipse/jetty/docs/programming/server/http/HTTPServerDocs.java[tags=movedContextHandler]
 ----
 
+[[handler-use-path-mappings]]
+==== PathMappingsHandler
+
+A web application may want to split the handling of HTTP requests so that they are handled by different ``Handler``s, to separate different web application concerns.
+
```

Comment:
Insert a paragraph here about how at a course level, this is typically done with ContextHandlers and context paths - splitting request handling into handlers that share code base, resource base etc.  link to section about contexts and contextHandlerCollection

Then say that to to so at a finer scale you can use the PathMappingsHandler and continue.

Perhaps somewhere also say that the ConditionalHandler can be used for not path based separation.

Reply:
Done.